src/api/app/components/watchlist_icon_component.html.haml

```diff
@@ -1,8 +1,8 @@
 - if object_to_be_watched_in_watchlist?
   = link_to('#', class: 'btn btn-outline-secondary',
             title: "Remove #{watchable_type_text} from watchlist",
-            data: { toggle: 'modal',
-                    target: '#delete-item-from-watchlist-modal',
+            data: { 'bs-toggle': 'modal',
+                    'bs-target': '#delete-item-from-watchlist-modal',
```

Comment:
This is probably something which worked before the Bootstrap 5 update. Anyway, this is documented upstream: https://getbootstrap.com/docs/5.2/components/modal/#toggle